azerothcore/azerothcore-wotlk
# fix(DB): Wrong Dialog for 1st Aid trainers

## Changes Proposed:
-  Adding conditions for First Aid trainer's dialogues

## Issues Addressed:
- Closes #4458



## SOURCE:

The linked issue and chromiecraft's issue have all the evidence.

## Tests Performed:

- Tested and working on Windows 10


## How to Test the Changes:

**Horde**
.go c 33780
Talk and see his presentation gossip
.quest add 6622
.quest complete 6622
Deliver the quest and talk to him again, having a different gossip.

**Alliance**
.go c 14520
Talk and see his presentation gossip
.quest add 6624
.quest complete 6624
Deliver the quest and talk to him again, having a different gossip.

## Known Issues and TODO List:

Complete


## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**, For example,**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
